engines/director/detection_tables.h

```diff
@@ -2203,7 +2212,8 @@ static const DirectorGameDescription gameDescriptions[] = {
 	MACDEMO1("provektormed", "Demo", "PVM_Demo", "7c80091b84d4d5d48f7c773b30b10598", 2025339, 311),
 
 	MACGAME1_l("pyramidint", "", "Pyramid Interactive", "f5277c53bacd27936158dd3867e587e2", 396919, Common::JA_JPN, 311),
-	MACDEMO1_l("pyramidint", "Demo", "PyramidInteractive", "f5277c53bacd27936158dd3867e587e2", 1591182, Common::JA_JPN, 311),
+	MACDEMO1_l("pyramidint", "Demo", "PyramidInteractive", "f5277c53bacd27936158dd3867e587e2", 1590926, Common::JA_JPN, 311),
+	MACDEMO1_l("pyramidint", "Demo", "Pyramid Interactive", "17c6ca896a3db4ee4de34708c9d2332c", 1198665, Common::JA_JPN, 311),
```

Comment:
The demo that we have on the website does have a file that matches this exactly, but it is a movie (MMDR). The projector (MMPB) has no space in the filename.

Reply:
Looks like I might have chosen the wrong file. Now that you fixed the filesize on the other one, it's detected. I'll just remove this line.